### ISSUE TITLE
Key Generation Wizard UX improvement

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#7298]
### Describe your problem(s)

Token Type  options are not visible, 

![image](https://user-images.githubusercontent.com/20179540/72778027-a0670880-3c3d-11ea-9572-c8cc44d0b33e.png)

### Describe your solution
Since the number of options are low its better to always display them with radio buttons.

### How will you implement it
Use radio buttons instead dropdown for options less than 6

**Comment by chanaka3d** [comment]
As of 3.2 release the toke type is only JWT we can close this issue.
